### PR TITLE
feat: captcha v2

### PR DESCRIPTION
**Pull request by nguyenhuudungz**

### CODE REVIEW

**Review by XiaoYhun — APPROVED** [review]
lgtm

File: src/index.tsx [hunk 1]
```diff
@@ -87,6 +87,10 @@ const hideLoader = () => {
   }, 100)
 }
 
+window.recaptchaOptions = {
```
Comment (nguyenhoaidanh) [inline src/index.tsx:90]: remove if unused

File: src/pages/Campaign/CampaignActions.tsx [hunk 2]
```diff
@@ -1,6 +1,10 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
```
Comment (nguyenhoaidanh) [inline src/pages/Campaign/CampaignActions.tsx:1]: remove it,  I don't see any error when remove it.

File: src/react-app-env.d.ts [hunk 3]
```diff
@@ -26,6 +26,7 @@ interface Window {
   dataLayer?: any[]
   chrome?: any
   opr?: any
+  recaptchaOptions?: any
```
Comment (nguyenhoaidanh) [inline src/react-app-env.d.ts:29]: remove if unused

**Review by nguyenhoaidanh — APPROVED** [review]

**Review by nguyenhoaidanh — DISMISSED** [review]

**Review by XiaoYhun — DISMISSED** [review]

**Review by nguyenhoaidanh — APPROVED** [review]

**Review by hungdoansy — APPROVED** [review]

**Pull request merged**